src/gf.c

```diff
@@ -2402,10 +2402,16 @@ jl_code_instance_t *jl_compile_method_internal(jl_method_instance_t *mi, size_t
                         /* specptr_matches_invokeptr */ 0,
                         /* from_image */ 0
                     } };
```

Comment:
this should be named `pending_flag`, since this bit pattern does not mean the `specialized_flag` is set